lib/src/package_analyzer.dart

```diff
@@ -255,6 +256,7 @@ class PackageAnalyzer {
       errorMessage = context.errors.map((e) => e.trim()).join('\n\n');
     }
     return Summary(
+      createdAt: DateTime.now(),
```

Comment:
```suggestion
      createdAt: DateTime.now().toUtc(),
```

otherwise it won't end with `Z` when render toJson.